### PR TITLE
Fixes in chapter 11 onwards [finished committing]

### PR DESCRIPTION
**Pull request by matanhakim**
Fixed two duplicates.

### CODE REVIEW

File: dependencies-in-practice.Rmd [hunk 1]
```diff
@@ -21,7 +21,7 @@ We're finally ready to talk about how to use different types of dependencies wit
 
 -   in your functions, below `R/`
 -   in your tests, below `tests/testthat/`
--   in your examples, in the help topics below `man/`
+-   in your examples and help topics, below `man/`
```
Comment (jennybc) [inline dependencies-in-practice.Rmd]: I'm reverting this change.

File: dependencies-mindset-background.Rmd [hunk 2]
```diff
@@ -255,7 +255,7 @@ Without knowing it, you've probably already used namespaces.
 Have you ever used the `::` operator?
 It disambiguates functions with the same name.
 For example, both the lubridate and here packages provide a `here()` function.
-If you attach lubridate, then here, `here()` will refer to the here version, because the last package attached wins.
+If you attach lubridate, then `here()` will refer to the here version, because the last package attached wins.
```
Comment (jennybc) [inline dependencies-mindset-background.Rmd]: I'm reverting this.

File: license.Rmd [hunk 3]
```diff
@@ -184,7 +184,7 @@ This is explicit in the [GitHub terms of service](https://docs.github.com/en/git
 
 [^license-4]: Some particularly risk averse organisations require contributors to provide a [developer certificate of origin](https://developercertificate.org), but this is relatively rare in general, and I haven't seen it in the R community.
 
-However, the author retains copyright of their code, which means that you can't change the license without their permission (more on that shortly).
+However, the author retains copyright of their code, which means that you can't change the license without their permission (more on that above).
```
Comment (jennybc) [inline license.Rmd]: I'm reverting this.

File: testing-design.Rmd [hunk 4]
```diff
@@ -629,7 +629,7 @@ test_that("foofy() does this", {
 
 ### Where to write files during testing {#sec-tests-files-where-write}
 
-If it's easy to avoid writing files from you tests, that is definitely the best plan.
+If it's easy to avoid writing files from you test, that is definitely the best plan.
```
Comment (jennybc) [inline testing-design.Rmd]: This needed more work.

**Review by jennybc — COMMENTED** [review]
Thanks! This was a mix of changes I could accept and others I could not. In the future, it's easier on the PR recipient if you split stuff like this into multiple PRs. It feels funny to make lots of little PRs, it makes many of them easier to just merge.

**Comment by matanhakim** [comment]
Thanks for your insights!
Promise to do better in the future 🙏

**Comment by jennybc** [comment]
Thanks again! Lots of these were good catches.

**Pull request merged**